docs/topics/forms/modelforms.txt

```diff
@@ -1230,10 +1230,53 @@ of a model. Here's how you can do that::
 
 Notice how we pass ``instance`` in both the ``POST`` and ``GET`` cases.
 
+Inline formsets and ``related_name``
+------------------------------------
+
+If the foreign key on your child model has a ``related_name`` to the parent,
+this is set to the formset's prefix when using ``inlineformset_factory``.
```

Comment:
I find this sentence a little confusing - the logic seems backwards. You write "this [the related_name] is set to the formset's prefix", when actually the formset's prefix is the thing being assigned.

I might reword this as "If the foreign key on your child model has a related_name to the parent, the formset uses this name as a prefix."

I'm not sure if it's necessary to specify "when using `inlineformset_factory`", since I think that's implicit in using inline formsets.